sky/clouds/cloud.py

```diff
@@ -470,7 +475,46 @@ def check_disk_tier_enabled(cls, instance_type: str,
         raise NotImplementedError
 
     @classmethod
-    # pylint: disable=unused-argument
+    def _check_instance_type_accelerators_combination(
+            cls, resources: 'resources_lib.Resources') -> None:
+        """Errors out if the accelerator is not supported by the instance type.
+
+        This function is overridden by GCP for host-accelerator logic.
+
+        Raises:
+            ResourcesMismatchError: If the accelerator is not supported.
+        """
+        assert resources.is_launchable(), resources
+
+        def _equal_accelerators(acc_requested, acc_from_instance_type):
+            if acc_requested is None:
+                return acc_from_instance_type is None
+            if acc_from_instance_type is None:
+                return False
+
+            for acc in acc_requested:
+                if acc not in acc_from_instance_type:
+                    return False
+                if acc_requested[acc] != acc_from_instance_type[acc]:
+                    return False
```

Comment:
IIUC, here we are requiring that the requested number of accelerators should match the amount the VM has, right? I think it's worth a comment.

Reply:
Good point! Yes, added a docstr for the method.